cumulusci/tasks/bulkdata/tests/tests/test_generate_from_snowfakery_task.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+from pathlib import Path
+
+from cumulusci.tasks.bulkdata.tests.test_bulkdata import _make_task, TaskOptionsError
+from tempfile import NamedTemporaryFile
+
+import yaml
+from sqlalchemy import create_engine
+
+try:
+    import snowfakery
+except ImportError:
+    snowfakery = None
```

Comment:
I think you could use `pytest.importorskip`: http://doc.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency